Eatery/Controllers/EateriesViewController.swift

```diff
@@ -209,6 +210,7 @@ class EateriesViewController: UIViewController, MenuButtonsDelegate, CLLocationM
     }
     
     func loadData(force: Bool, completion:(() -> Void)?) {
+        if initialLoad && !force { return }
```

Comment:
Guard statement: 
`guard force || !initialLoad`

Reply:
This whole method is kind of strange actually. Having an initial load method would be more clear.